src/components/PostList/PostList.tsx

```diff
@@ -1,5 +1,20 @@
 import React from 'react';
+import { Post } from '../../types/types';
+import { PostInfo } from '../PostInfo';
+import './PostList.scss';
 
-export const PostList: React.FC = () => (
-  <>Put the list here</>
+interface Props {
+  list: Post[];
+}
+
+export const PostList: React.FC<Props> = ({ list }) => (
+  <ul className="postList">
+    {list.map(
+      post => (
+        <li>
+          <PostInfo {...post} key={post.id} />
```

Comment:
Move the `key` prop to the `li` element for better performance and to avoid React warnings. 🚀